demos-and-sample-apps/sdk-browser-overview.md

```diff
@@ -8,46 +8,46 @@ position: 0
 
 # Telerik UI for Xamarin Demo Applications
 
-**Telerik UI for Xamarin** provides an easy-to-use infrastructure with many fully featured examples demonstrating our Xamarin controls. You can review the source code of each example and get familiar with the configuration possibilities that each provides.
+**Telerik UI for Xamarin** provides an easy-to-use infrastructure with many fully-featured examples demonstrating our Xamarin controls. You can review the source code of each example and get familiar with the configuration possibilities that each provides.
 
-There are two sets of demos showing Telerik Xamarin.Forms controls: **Telerik UI for Xamarin Samples** application and **SDK Browser** application. In this topic you can read what is the purpose of each and how they both are useful.
+There are two sets of demos showing Telerik Xamarin.Forms controls: the **Telerik UI for Xamarin Samples** application and the **SDK Browser** application. In this topic, we describe how to use them and explain their differences.
 	
 ## Telerik UI for Xamarin Samples Application
```

Comment:
Can we add more details on what to do with the Xamarin Samples Application:
- How to run it on PC? 
- Any prerequisites?

Are the three ways to access the samples app identical? It looks like option 1 and 2 allow you to directly review the code. But what about option 3 - installing the app? We may need to point out the differences.

Reply:
this is common knowledge for Xamarin users